CITATION.cff

```diff
@@ -0,0 +1,34 @@
+cff-version: 1.2.0
+title: >-
+  s2fft: Differentiable and accelerated spherical transforms with JAX
+message: >-
+  If you use this software, please cite it using the metadata from this file.
+type: software
+authors:
+  - given-names: Matthew A.
+    family-names: Price
+    affiliation: University College London
+  - given-names: Jason
+    orcid: 'https://orcid.org/0000-0002-5852-8890'
```

Comment:
I'm not actively using this so no need to cite.  If anything would be better to cite my website (www.jasonmcewen.org)

Reply:
I've added your website here and in preferred citation (and also Matt's). Unless you have a strong preference of not including ORCiD I would say better to include it as it helps with having research outputs automatically indexed - for example if we start having releases archived on Zenodo using the [Zenodo GitHub integration](https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content) then your ORCiD will be automatically pulled from the `CITATION.cff` file and added to the metadata for the archive on Zenodo. As your ORCiD is linked to your UCL profile / entry on RPS this means any archive on Zenodo can then also be automatically synced to your RPS profile.

Reply:
Ok, fine to keep the ORCiD then.